admin-dev/themes/default/js/tree.js

```diff
@@ -22,7 +22,7 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
-const Tree = function (element, options) {
+var Tree = function (element, options) {
```

Comment:
```suggestion
const Tree = function (element, options) {
```

Reply:
that's the fix :D. (I explained in the PR description, I'll make a comment so that the eslint doesn't complain)

Reply:
In this case, use `let`. And remove the comment

Reply:
`let` triggers the same error, it doesn't allow declaring the same variable twice

Reply:
Ok. In this case, may be @NeOMakinG can help you fixing correctly this bug ?

Reply:
well we can do it in 1.7.9 if we want to, I'll create an issue based on this one.  This behavior is not a 1.7.8 regression

Reply:
I'll try this 👍  I didn't know global would be considered better than `var` by the linter

Reply:
I've a doubt about window for this, but global may do the job way better